docs/advanced_topics/i18n.rst

```diff
@@ -0,0 +1,49 @@
+====================	
+Internationalisation	
+====================	
+
+Multi-language content
+======================
+
+In its basic configuration, Wagtail does not provide specific support for multi-language content. This is because there is no single preferred approach to handling translations that works for all scenarios - various approaches are possible, depending on factors such as:
+
+* The number of languages you intend to support
+* Whether there is a 
```

Comment:
Missing content here?

Reply:
Oops! Fixed in 37cc64d4fac912daeeebb623df60017a1780a755.